docs/5-build-experiment.rst

```diff
@@ -31,3 +26,7 @@ which will build the source code and set up the following workspace directory st
                                 <problem>/
                                     <benchmark>_<ProgrammingModel>_<problem>
                                         execute_experiment
+
+
+If you change any build instructions, rerun the ``ramble -P -D . workspace setup`` command
+to re-setup the workspace and rebuild the binaries as necessary.
```

Comment:
Perhaps reference the FAQ added in this PR? (as it is, "build instructions" could mean a few different things)

Reply:
yes, good catch.